torch_tvm/compiler.h

```diff
@@ -67,13 +69,20 @@ struct TVMCompiler {
   std::string device_type_;
   std::string device_;
   std::string host_;
+  int device_id_;
   tvm::runtime::Module build_mod_;
   DebugLogger debug_logger_;
   // Not used in OSS
   at::Tensor activation_buffer_;
   std::vector<int64_t> activation_buffer_shape_;
   int64_t max_activation_buffer_size_{0};
 
+  std::string handle_str_;
+  std::unique_ptr<torch::jit::InterpreterState> fallback_state_;
+  std::unordered_set<torch::jit::CompleteArgumentSpec> bad_specs_;
```

Comment:
nit: add a comment explaining the purpose of this param -> to cache uncompiled-able specs

Reply:
yep